timers.js

```diff
@@ -1,4 +1,92 @@
+/******************************************
+BONUS VERSION WITH JQUERY
+******************************************/
 
+// how do I change my global variables in init to local variables that everything in the timer object can access?
 
+var timer = {
+  init: function() {
+    stopWatch = $("#timer");
+    reset = $("#reset");
+    start = $("#start");
+    pause = $("#pause");
+    timerStatus = "off";
+    seconds = 0;
+    start.on("click", function () {
+      timer.startTimer();
```

Comment:
Nothing wrong with using `timer` here, but good practice is to use `this` instead of the object variable name. The benefit of this will be clear later on in the course when we talk more about object-oriented programming.